### DISCUSSION TITLE
Getting Error

### DISCUSSION THREAD

**Discussion by AbhayPatel98** [Cyfrin/foundry-full-course-cu#1676] [General]
No tests found in project! Forge looks for functions that starts with `test`.

<img width="589" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/114071293/6b2eadd9-2853-4550-8202-018df0e19601">

any solution?

**Comment by EngrPips** [comment]
Hello @AbhayPatel98, Please show your test file.
